package-lock.json

```diff
@@ -3,6571 +3,10 @@
   "requires": true,
   "lockfileVersion": 1,
   "dependencies": {
-    "@evocateur/libnpmaccess": {
```

Comment:
The fact that 6569 lines are *deleted* from root `package-lock.json` is suspicious? (Why the heck do we have so many dependencies anyway?).

Reply:
True. This is VERY strange. There shouldn't be much in the root node_modules. How did they all get there?

Reply:
Turns out lerna has "dependency issues".